server/dive_server/viame.py

```diff
@@ -467,25 +460,11 @@ def postprocess(self, folder, skipJobs):
         .errorResponse('Using a reserved metadata key', 400)
     )
     def update_metadata(self, folder, data):
-        expunged = []
-        meta = {}
-        # filter data for only allowed metadataMutable and error on Reserved
-        for key in data.keys():
-            if key in metadataReserved:
-                raise RestException(
-                    f'Using a reserved metadata key: {key}, please remove it and use the proper endpoint'
-                )
-            elif key not in metadataMutable:
-                expunged.append(key)
-            else:
-                meta[key] = data[key]
-
-        folder['meta'].update(meta)
+        validated = models.MetadataMutableUpdate(**data)
+        for name, value in validated.dict(exclude_none=True).items():
```

Comment:
`exclude_none` prevents overwriting the existing metadata if the key isn't part of the update payload.